diff --git a/src/chains/mudFoundry.ts b/src/chains/mudFoundry.ts
index 29705d4684c042eed589408ba190e7f59b98eef5..7ed53829217222e005ab818c672b5365c5454153 100644
--- a/src/chains/mudFoundry.ts
+++ b/src/chains/mudFoundry.ts
@@ -1,7 +1,7 @@
 import { foundry } from "viem/chains";
 import { MUDChain } from "./types";
 
-export const mudFoundry = {
+export const mudFoundry: MUDChain = {
   ...foundry,
   fees: {
     // This is intentionally defined as a function as a workaround for https://github.com/wagmi-dev/viem/pull/1280
